SPECS/asc/asc.spec

```diff
@@ -0,0 +1,24 @@
+Summary:        Package for Mariner to meet Azure Security Baseline 
+Name:           asc
+Version:        1.0
+Release:        1%{?dist}
+License:        MIT
+Requires:       filesystem-asc
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Base
+URL:            https://aka.ms/mariner
+
+%description
+Package for Mariner to meet Azure Security Baseline by adding multiple config files in /etc/modprobe.d
+
+%prep
+
+%build
+
+%files
```

Comment:
What's the point of an empty package?

Reply:
The main goal of package is to make filesystem-asc sub-package as its dependency (line Requires: filesystem-asc). The user has the option to install asc package which requires filesystem-asc to be installed. Filesystem-asc is to meet some asc rules but we might not want it in default.

Reply:
We are looking to make a series of similar compliance/security packages. In the future they will likely also include systemd services to remediate any issues automatically.

Reply:
@dmcilvaney: are you saying that we already know `asc.spec` will be extended and will actually install something in the future and we're just preparing the ground for that in this PR? If that's the case, then I'm fine with this spec the way it is right now. If we're not 100% we will add anything to this package, however, then I'm not sure if we should be creating at the moment.